_about_livecoms/02_thanks.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: single
+sidebar:
+  nav: thanks.md
+title: Thanks
+excerpt: Thanks for special services in helping make LiveCoMS possible
+permalink: /about/thanks/
+---
+
+LiveCoMS wishes to give special thanks to:
+
+* CU Boulder Libraries, for their generous donation of seed money to start the journal.
+
+* The CU Boulder Department of Chemical and Biological Engineering for
+  agreeing to help serve as publisher, and providing logistical support
```

Comment:
@mrshirts is your dept the publisher?  I would have thought it's just CU.  If so, just break this into two bullets

Reply:
Cleaned up.